.github/workflows/ci.yml

```diff
@@ -51,13 +48,3 @@ jobs:
           toolchain: stable
           components: rustfmt
       - run: cargo fmt --check
-
-  # https://github.com/rust-lang/crater/blob/9ab6f9697c901c4a44025cf0a39b73ad5b37d198/.github/workflows/bors.yml#L125-L149
-  end-success:
-    name: bors build finished
-    if: success()
-    runs-on: ubuntu-latest
-    needs: [check, test]
-    steps:
-      - name: Mark the job as successful
-        run: exit 0
```

Comment:
Actually, we might need to add this back in (albeit with a slightly different name), so we can say /this/ is the status check that must pass for something to make it out of the merge queue successfully.